File: 2015/02/1.rb
feet = 0

File.open('input.txt').each do |line|
  l, w, h = line.split('x').map(&:to_i)
  feet += 2*l*w + 2*w*h + 2*h*l + [l*w, w*h, h*l].min
end

puts feet

File: 2015/02/1.py
feet: int = 0

with open("input.txt") as f:
    for line in f:
        l, w, h = map(int, line.split('x'))

        feet += 2*l*w
        feet += 2*w*h
        feet += 2*h*l
        feet += min(l*w, w*h, h*l)

print(feet)
